src/App.js

```diff
@@ -0,0 +1,23 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './App.css';
+import NavBar from './components/NavBar';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import NotFound from './components/NotFound';
+
+function App() {
+  return (
+    <Router>
+      <NavBar />
+      <section className="content">
+        <Routes>
+          <Route path="/bookstore" element={<Books />} />
```

Comment:
- [ ] Kindy make sure  this `Books` component is rendered at the root route of you App(`/`)

```js
          <Route path="/" exact element={<Books />} />

```